prep-css-syntax/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+  <style>
+    h1 {font-family: Arial, Helvetica, sans-serif;}
+    .p1 {font-size: 2rem;}
+    h3 {color:firebrick}
+    h2 {font-style: italic;}
+  </style>
+</head>
+<body>
+  <h1>This h1 tag has been given Arial as a font-family.</h1>
+  <p>I am a p tag which has a font-size of 2rem!!!</p>
```

Comment:
To target this paragraph you'll either need a `p` element type selector or add a class attribute of `"p1"` to match the selector in your CSS. If it works, the text will be much larger.